src/viewer_window.cpp

```diff
@@ -345,6 +345,20 @@ class DatarefViewerWindow {
 		XPSetWidgetProperty(scroll_bar, xpProperty_ScrollBarMin, 0);
 		XPSetWidgetProperty(scroll_bar, xpProperty_ScrollBarMax, 0);
 
+        FILE * pFile;
+        pFile = fopen ("./Resources/plugins/DataRefTool/drtpref.txt","r+");
+        int tmp_last_case_sensitive = last_case_sensitive;
+        int tmp_last_regex = last_regex;
+        int tmp_last_changed = last_changed;
+
+        if (pFile) {
+           fscanf(pFile,"%d %d %d %d %d %d %d", &last_left, &last_top, &last_right, &last_bottom, &tmp_last_case_sensitive, &tmp_last_regex, &tmp_last_changed);
```

Comment:
What if the file isn't in the format we expect? We should check the fscanf return value.